scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2638,23 +2575,21 @@ u64 antistall_set(u64 dsq_id, u64 jiffies_now)
  */
 static bool antistall_scan(void)
 {
-	s32 cpu;
-	u64 dsq_id;
+	s32 llc;
+	u64 layer_id;
 	u64 jiffies_now;
 
 	if (!enable_antistall)
 		return true;
 
 	jiffies_now = bpf_jiffies64();
 
-	bpf_for(dsq_id, 0, nr_layers) {
-		antistall_set(dsq_id, jiffies_now);
-	}
```

Comment:
This loop was buggy as it wouldn't visit all layer DSQs when topology is enabled.